### PR TITLE
Move TitusMaster leader resolve to titus-runtime + add skeleton task-relocation component implementation

### PR DESCRIPTION
**Pull request by tbak**

### CODE REVIEW

File: titus-supplementary-component/task-relocation/src/main/resources/laptop.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# set the webapp resource base to / since webapp doesn't exist when not packaged as a war
+governator.jetty.embedded.webAppResourceBase=/
+governator.jetty.embedded.port=7020
+
+titus.federation.endpoint.grpcPort=7021
```
Comment (corindwyer) [inline titus-supplementary-component/task-relocation/src/main/resources/laptop.properties]: is this needed?

File: titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/master/DefaultTitusMasterOperations.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+public class DefaultTitusMasterOperations implements TitusMasterOperations {
+
+    private static final Logger logger = LoggerFactory.getLogger(DefaultTitusMasterOperations.class);
+
+    private final JobCacheResolver jobCacheResolver;
+
+    private final Subscription subscription;
+
+    @Inject
+    public DefaultTitusMasterOperations(JobCacheResolver jobCacheResolver) {
```
Comment (corindwyer) [inline titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/master/DefaultTitusMasterOperations.java]: Guessing this is a empty implementation for now?

File: titus-supplementary-component/task-relocation/src/main/resources/laptop.properties (hunk 1 above)
Comment (tbak) [reply titus-supplementary-component/task-relocation/src/main/resources/laptop.properties]: No, I will remove this in the next PR.

File: titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/master/DefaultTitusMasterOperations.java (hunk 2 above)
Comment (tbak) [reply titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/master/DefaultTitusMasterOperations.java]: Yes, just to log something.

**Pull request merged**